src/getHumanAge.test.js

```diff
@@ -7,4 +7,52 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return a correct dog and cats years for an old animals', () => {
+    const result = getHumanAge(100, 100);
+
+    expect(result).toEqual([21, 17]);
+  });
+
+  test('should return zero for 0 array', () => {
+    const result = getHumanAge(0, 0);
+
+    expect(result).toEqual([0, 0]);
+  });
+
+  test('should return zero for animals younger 15 years', () => {
+    const result = getHumanAge(14, 14);
+
+    expect(result).toEqual([0, 0]);
+  });
+
+  test('should return 1 for animals exactly 15 years', () => {
+    const result = getHumanAge(15, 15);
+
+    expect(result).toEqual([1, 1]);
+  });
+
+  test('should return 1 for animals between 15 and 24 years', () => {
+    const result = getHumanAge(23, 23);
+
+    expect(result).toEqual([1, 1]);
+  });
+
+  test('should return 2 for animals 24 y.o. ', () => {
+    const result = getHumanAge(24, 24);
+
+    expect(result).toEqual([2, 2]);
+  });
+
```

Comment:
The test description 'should return 2 for animals between 24 & 28 and 24 % 25 years' is unclear. Consider rephrasing it to something like 'should return 2 for animals aged between 24 and 28 years'.